### PR TITLE
CRUD of company management

### PR DESCRIPTION
**Pull request by mild-tn**
- Add Carbon dependency 
- Add CompanyController file 
- Add Company repository
- Add model: Address, Company, and MOU
- Add UUID config file (Traits)
- Change charset to utf8 
- Update AppserviceProvider 
- Add API route for crud
- Update migrate file
- I wrote unittest. 

Related card: https://trello.com/c/956AzBA3/24-%F0%9F%92%9C-sp3-crud-of-company-manage and https://trello.com/c/kSsRTbfG/25-%F0%9F%92%9C-sp3-api-for-management-company

### CODE REVIEW

**Comment by mild-tn** [comment]
[💜 [SP3] API for management company](https://trello.com/c/kSsRTbfG/25-%F0%9F%92%9C-sp3-api-for-management-company)

**Comment by mild-tn** [comment]
[💜 [SP3] API for management company](https://trello.com/c/kSsRTbfG/25-%F0%9F%92%9C-sp3-api-for-management-company)

**Comment by mild-tn** [comment]
[💜 [SP3] CRUD of company manage](https://trello.com/c/956AzBA3/24-%F0%9F%92%9C-sp3-crud-of-company-manage)

File: database/migrations/2020_11_18_143751_addresses_table.php [hunk 1]
```diff
@@ -19,11 +19,13 @@ public function up()
             $table->string('address_two');
             $table->string('lane');
             $table->string('road');
-            $table->string('sub-district');
+            $table->string('sub_district');
             $table->string('district');
             $table->string('province');
-            $table->string('postal_code');
+            $table->string('postal_code', 10);
```
Comment (kaotu) [inline database/migrations/2020_11_18_143751_addresses_table.php:25]: อันนี้จะให้รหัสไปรษณีย์มันพอดีเลยมะ ของไทยมันมี 5 หลักอ่ะ

Comment (mild-tn) [reply database/migrations/2020_11_18_143751_addresses_table.php:25]: ด้าย ตอนใส่ก็แค่เผื่อๆ ไว่หน่ะ ถ้าพอดีก็ได้

**Review by kaotu — APPROVED** [review]

**Review by Jirattikarn — APPROVED** [review]

**Review by kaotu — APPROVED** [review]

**Comment by mild-tn** [comment]
[💜 [SP3] จัดการข้อมูลบริษัท (company management page)](https://trello.com/c/HuMxU6sj/26-%F0%9F%92%9C-sp3-%E0%B8%88%E0%B8%B1%E0%B8%94%E0%B8%81%E0%B8%B2%E0%B8%A3%E0%B8%82%E0%B9%89%E0%B8%AD%E0%B8%A1%E0%B8%B9%E0%B8%A5%E0%B8%9A%E0%B8%A3%E0%B8%B4%E0%B8%A9%E0%B8%B1%E0%B8%97-company-management-page)

**Pull request merged**